scripts/deploy.sh

```diff
@@ -90,9 +91,14 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
       echo "osd_directory: true" | tee -a $RPCD_VARS
       echo "osd_directories:" | tee -a $RPCD_VARS
       echo "  - /var/lib/ceph/osd/mydir1" | tee -a $RPCD_VARS
-      sed -i "s/glance_default_store:.*/glance_default_store: rbd/" /etc/openstack_deploy/user_variables.yml
-      echo "nova_libvirt_images_rbd_pool: vms" | tee -a /etc/openstack_deploy/user_variables.yml
-      echo "cinder_ceph_client_uuid:"  | tee -a /etc/openstack_deploy/user_secrets.yml
+      echo "glance_default_store: rbd" | tee -a /etc/openstack_deploy/user_osa_variables_defaults.yml
```

Comment:
This is for an AIO, support do not deploy customers using the AIO code. This code is essentially providing the function of support by determining a set of vars that should be set for the deployment, because it's an AIO and because it's always the same it can be automated.